### PR TITLE
Just a tiny fix that slowly became QoL Patch IX: Burenya

### PR DESCRIPTION
**Pull request by ThePainkiller**
## About The Pull Request

**EDIT:** At this point this might as well be QoL patch 9...

- Fixed sugar icon state always defaulting to the small shaker
- Added Salt bag sprite, properly pathed
- Added better jelly donuts sprites
- Moves one of the Vox on the Swamp House to the bathroom to prevent it killing the farm animals 100% of the time (not guaranteed the other one won't). Also adds in a small power cell for the electrolyzer as it suddenly started spawning without one making it useless.
- Adds a coffee machine to the Greyson Offices in the "staff meeting room" as well as a mug and sugar. Let's assume a qt Greyson positronic waifu poured coffee for important people on their meetings at the large table.
- Adds a butter slice as a requirement for all pie recipes
- Adds a mechfab to the end of Prepper's vault
- Adds Strawberry icecream sandwich (use a cookie on a strawberry icecream to craft)
- Adds a tube of Nanopaste to the Hunting Lodge's medical area for those hunters that died a long time ago
- ~~Makes ablative vest into full body ablative vest ([Eris port](https://github.com/discordia-space/CEV-Eris/pull/5893/))~~ nope
- Removes duplicate "Robot" from robot component's name var so that those aren't named "robot robot" in game
- Fixes burn barrel transforming into a microwave when broken, adds sprites for its broken and dirty states
image

- Adds a cursed plushie that drinks your Claw energy drinks and nya's at you when poked.
arcueid

- Ports Eris' [Warrant Projector](https://github.com/discordia-space/CEV-Eris/pull/4840), with animated sprite. Modifies departmental names where applicable for setting cohesion and also the warrant program on computers/PDA to accomodate for its template.
- Fixes Fine Nicotine by making it give its desired effects on "ingestion" (aka smoking normally) rather than overdose.

## Changelog
:cl:
add: Adds a coffee machine to the Greyson Offices, a mech fabricator to the end of Prepper's Vault (near the AI vault), and a tube of Nanopaste to the Hunter's Lodge
add: Added Strawberry Icecream Sandwich. Use a Cookie on a Strawberry Icecream to craft.
add: Added a cursed catgirl plushie that nya's when poked. It's on the random pools for misc and large plushies.
add: Ports Eris's (actually Bay's) Warrant Projector, with animated sprite. Use in-hand to load a warrant, attack people with them to show them the warrant. Also modifies the bounty program on computer/PDA to better accomodate for the warrant projector's template.
tweak: All pies (Banana, cherry, etc) now require 1 butter slice to make.
tweak: Moves one of the Vox-like birds from the main swamp house into the bathroom area since they keep killing the cow and chicken with their rocks. If it doesn't help keeping them alive I'll just remake the pen entirely.
fix: Fixed sugar bag icon state always defaulting to the small sugar shaker
fix: Robot components should no longer be named "robot robot <component_name>"
fix: Fine nicotine should now properly apply its effects when smoked instead of overdosed upon or injected
imageadd: Added Salt Bag sprite and better jelly donuts sprites, adds icon states for dirty and broken burn barrels, adds 1 fumo plushie everyone will hate
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -327,11 +327,12 @@
 	icon_state = "bulletproof_ironhammer"
 
 /obj/item/clothing/suit/armor/laserproof
-	name = "ablative armor vest"
+	name = "full ablative armor vest"
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/armor.dm:330]: Denied. You can get gloves/boots

**Comment by Trilbyspaceclone** [comment]
Get a better bag of salt sprite as well

File: code/modules/clothing/suits/armor.dm (hunk 1 above)
Comment (ThePainkiller) [reply code/modules/clothing/suits/armor.dm:330]: It's an eris port but I assume it's because of the ablative gloves/boots from greyson right?

Comment (Trilbyspaceclone) [reply code/modules/clothing/suits/armor.dm:330]: Iron lock so ya graysons, but every cargopack/disk/kit has full sets of laser proofing.
only time you dont get full set is if you rng role

**Comment by ThePainkiller** [comment]
> Get a better bag of salt sprite as well

Done, did the best I could myself!

**Review by Trilbyspaceclone — APPROVED** [review]

**Pull request merged**